api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java

```diff
@@ -545,20 +565,23 @@ public static SqlCohortDefinition getMQ15DenB1(
             + "                                                  ON e.encounter_id = o.encounter_id "
             + "                                   WHERE  p.voided = 0 "
             + "                                          AND e.voided = 0 "
+            + "                                          AND o.voided = 0 "
             + "                                          AND e.location_id = :location "
             + "                                          AND e.encounter_type = ${6} "
             + "                                          AND o.concept_id = ${23739} "
             + "                                          AND o.value_coded = ${23720} "
-            + "                                          AND e.encounter_datetime <= :endDate "
+            + "                                          AND e.encounter_datetime BETWEEN Date_sub(:revisionEndDate, INTERVAL 14 month) "
+            + "                                          AND Date_sub(:revisionEndDate, INTERVAL 11 month) "
             + "                                   GROUP  BY p.patient_id) a3 "
             + "                       ON p.patient_id = a3.patient_id "
             + "                WHERE  p.voided = 0 "
             + "                       AND e.voided = 0 "
+            + "                       AND o.voided = 0 "
             + "                       AND e.location_id = :location "
             + "                       AND e.encounter_type = ${6} "
             + "                       AND o.concept_id = ${23739} "
             + "                       AND o.value_coded <> ${23720} "
-            + "                       AND e.encounter_datetime <= :endDate "
+            + "                       AND e.encounter_datetime <= :revisionEndDate "
             + "                GROUP  BY p.patient_id) not_dt) b1; ";
```

Comment:
The inner semicolon is NOT required here

Reply:
Done removing the semicolon